pom.xml

```diff
@@ -659,9 +659,14 @@
         <artifactId>jackson-dataformat-smile</artifactId>
         <version>${jackson.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.dataformat</groupId>
+        <artifactId>jackson-dataformat-yaml</artifactId>
+        <version>${jackson.version}</version>
+      </dependency>
       <dependency>
         <groupId>com.fasterxml.jackson.jaxrs</groupId>
-        <artifactId>jackson-jaxrs-json-provider</artifactId>
+        <artifactId>jackson-jaxrs-base</artifactId>
```

Comment:
In the Jira, it was mentioned that jackson-jaxrs-base version needs to be updated and jackson-jaxrs-json-provider comes in the jackson-jaxrs-base library. So that's why I used jackson-jaxrs-base.

Reply:
It seems that you have not changed this pom yet: https://github.com/apache/hive/blob/master/testutils/ptest2/pom.xml